tests/sanity/ignore-2.13.txt

```diff
@@ -3,5 +3,4 @@ plugins/modules/cmci_get.py validate-modules:missing-gplv3-license # Licence is
 plugins/modules/cmci_action.py validate-modules:missing-gplv3-license # Licence is Apache-2.0
 plugins/modules/cmci_create.py validate-modules:missing-gplv3-license # Licence is Apache-2.0
 plugins/modules/cmci_delete.py validate-modules:missing-gplv3-license # Licence is Apache-2.0
-plugins/modules/cmci_update.py validate-modules:missing-gplv3-license # Licence is Apache-2.0
-plugins/doc_fragments/cmci.py pep8:E501 # Link can't be split or it does not render correctly
```

Comment:
Any idea why this was removed?

Reply:
So from what I've learnt, this disables the pep8 sanity test on that specific file. When that line is left in the file and the sanity tests are run:

```
ERROR: Found 1 pep8 issue(s) which need to be resolved:
ERROR: tests/sanity/ignore-2.13.txt:7:1: A100: Ignoring 'E501' on 'plugins/doc_fragments/cmci.py' is unnecessary
```

And when removing that line as it's considered 'unnecessary', the sanity tests pass with no problems...

I can show you these tests running in real time if that will be beneficial, but it's odd that this behaviour has changed since the sanity tests were written.

Reply:
Thanks for investigating! I trust that they're passing then. I expect that it's behaviour to do with an older version of ansible we were using.